seacrowd/sea_datasets/basaha_corpus/basaha_corpus.py

```diff
@@ -0,0 +1,210 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+BasahaCorpus contains short stories in four Central Philippine languages \
+    (Minasbate, Rinconada, Kinaray-a, and Hiligaynon) for low-resource \
+    readability assessment. Each dataset per language contains stories \
+    distributed over the first three grade levels (L1, L2, and L3) in \
+    the Philippine education context. The grade levels of the dataset \
+    have been provided by an expert from Let's Read Asia.
+"""
+
```

Comment:
Sorry forgot to comment, but, similar to the other PRs submitted, this line seems to be duplicate of the `_DESCRIPTION`, so let's remove this. Similar to the description in the bottom of class declaration (line 93-100), could you please rephrase into a short version of description instead of the one listed in `_DESCRIPTION`.

Reply:
I just remove the template header and simplify additional description on https://github.com/SEACrowd/seacrowd-datahub/pull/606/commits/e9946a2b521119178b3c922687a4f5808ec1b9a4.